emcc

```diff
@@ -109,7 +109,8 @@ stderr = PIPE if not DEBUG else None # unless we are in DEBUG mode
 EMCC_CXX = '--emscripten-cxx' in sys.argv
 sys.argv = filter(lambda x: x != '--emscripten-cxx', sys.argv)
 
-shared.check_sanity(force=DEBUG)
+if len(sys.argv) != 1 and '--help' not in sys.argv and sys.argv[1] != '--version':
+  shared.check_sanity(force=DEBUG)
```

Comment:
i have to tweak this a little, as several sanity checks are failing. I think we can run the checks on `emcc` with no arguments, but otherwise leave things as in this change.